cas-server-webapp/src/main/webapp/WEB-INF/casComponentsMapping.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:util="http://www.springframework.org/schema/util"
```

Comment:
Understood, thanks. That makes sense to me. 

My only other concern is the list of monitors. I do see a use case where people would want to add/remove monitors, and I'd have that available in the deployer file, rather than here. (Eventually, we could do away with monitors possibly and move onto actuators)